Resources/Prototypes/_DV/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -397,3 +397,39 @@
     sprite: _DV/Clothing/OuterClothing/Hardsuits/ERTSuits/eliteertjanitor.rsi
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHardsuitERTJanitorElite
+
+#LO Hardsuit
+- type: entity
+  parent: [ClothingOuterHardsuitSalvage, BaseJetpack, BaseCargoContraband]
+  id: ClothingOuterHardsuitRhino
+  name: rhino industrial hardsuit
+  description: A hardsuit adorned with purple fabric and gold metal. Made specifically for Nanotrasen for their Logistics Officers, this hardsuit boasts better protection than most salvage suits and has an internal jetpack.
```

By Nanotrasen, presumably?
```suggestion
  description: A hardsuit adorned with purple fabric and gold metal. Made specifically by Nanotrasen for their Logistics Officers, this hardsuit boasts better protection than most salvage suits and has an internal jetpack.
```